docs/source/routing/query-planning/native-query-planner.mdx

```diff
@@ -7,73 +7,50 @@ redirectFrom:
   - /router/executing-operations/native-query-planner
 ---
 
-<PreviewFeature />
-
-Learn to run the GraphOS Router with the Rust-native query planner and improve your query planning performance and scalability.
+Learn about the Rust-native query planner in GraphOS Router. The planner is GA as of v1.59.0.
 
 ## Background about query planner implementations
 
 In v1.49.0 the router introduced a [query planner](/graphos/routing/about-router#query-planning) implemented natively in Rust. This native query planner improves the overall performance and resource utilization of query planning. It exists alongside the legacy JavaScript implementation that uses the V8 JavaScript engine, and it will eventually replace the legacy implementation.
 
-### Comparing query planner implementations
-
-As part of the effort to ensure correctness and stability of the new query planner, starting in v1.53.0 the router enables both the new and legacy planners and runs them in parallel to compare their results by default. After their comparison, the router discards the native query planner's results and uses only the legacy planner to execute requests. The native query planner uses a single thread in the cold path of the router. It has a bounded queue of ten queries. If the queue is full, the router simply does not run the comparison to avoid excessive resource consumption.
+As of v1.59.0, the native query planner is the default planner in the router. As part of this, Deno, and by extension v8, are no longer intialised at router startup. The legacy implementation is deprecated, but it is still possible to opt-in to legacy query planner.
```

Comment:
```suggestion
As of v1.59.0, the native query planner is the default planner in the router. As part of this, Deno, and by extension v8, are no longer initialized at router startup. The legacy implementation is deprecated, but it is still possible to configure the router to run with the legacy query planner.
```

Reply:
ooops i need to set up an american spell checker for my .md files 😅